src/pages/University_search.js

```diff
@@ -60,8 +60,9 @@ export const UniversitySearch = () => {
             
           />
           <div id="autoFill" style={{ position: "absolute", width: "100%", top: "38px"}}>
-            {autoFillUniversity.map((universityName) => (
+            {autoFillUniversity.map((universityName, i) => (
               <button 
+                key = {i}
```

Comment:
Can we use the university name as a key? Autofill suggestions are unique based on input right?

Reply:
Good point I will change it